content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -76,6 +76,12 @@ Disables all CLI warnings generated by `logger.warn`.
 
 To configure `prisma generate` to [generate a Prisma Client for Data Proxy](/data-platform/classic-projects/data-proxy) on any execution, even when `--data-proxy` is not explicitly set for the CLI command, set the value of this environment value to a truthy value (e.g. `PRISMA_GENERATE_DATAPROXY=true`).
 
+<Admonition type="warning">
+
+**Deprecated**: `PRISMA_GENERATE_DATAPROXY` is deprecated in [5.2.0](https://github.com/prisma/prisma/releases/tag/5.2.0) as `--accelerate` and `--data-proxy` are no longer required to use Accelerate or Data Proxy. To get the same behavior as before, you can use the [`--no-engine` option](./command-reference#options-1) when generating your client.
```

Comment:
You can't really when this env var is necessary, which is durung `generate` triggered by `postinstall`.
Is that a problem @millsp?

Reply:
You can, but you need to use `PRISMA_GENERATE_NO_ENGINE`, worth adding to the PR

Reply:
I'll let @jharrell decide on that 😅

Reply:
Didn't know that `PRISMA_GENERATE_NO_ENGINE` was an option now. I'll add it.